lib/nifty_services/base_create_service.rb

```diff
@@ -24,86 +23,75 @@ def execute
     end
 
     private
+
     def save_record
       begin
         @record.save
       rescue => e
         on_save_record_error(e)
-        return false
+      ensure
+        @record.valid?
       end
     end
 
     def on_save_record_error(error)
-      return unprocessable_entity_error!(error)
+      unprocessable_entity_error!(error)
```

Comment:
@viniciuszago As I said, explicit return make code more clear in this case. Since